$NetBSD: patch-po_Makefile.in.in,v 1.1 2022/11/27 19:57:39 gutteridge Exp $

Remove directories after purging translation files. (During the build
phase, translation files are generated, installed, and then deleted.
This needs to be better sorted out upstream.)

+++ po/Makefile.in.in
@@ -367,6 +367,7 @@ uninstall-data-yes:
 	  for lc in LC_MESSAGES $(EXTRA_LOCALE_CATEGORIES); do \
 	    rm -f $(DESTDIR)$(localedir)/$$lang/$$lc/$(DOMAIN).mo; \
 	  done; \
+	  rm -fr $(DESTDIR)$(localedir); \
 	done
 
 check: all
